lightning-invoice/src/utils.rs

```diff
@@ -407,6 +594,31 @@ mod test {
 		assert_eq!(events.len(), 2);
 	}
 
+	#[test]
+	fn test_create_invoice_with_description_hash() {
+		let chanmon_cfgs = create_chanmon_cfgs(2);
+		let node_cfgs = create_node_cfgs(2, &chanmon_cfgs);
+		let node_chanmgrs = create_node_chanmgrs(2, &node_cfgs, &[None, None]);
+		let nodes = create_network(2, &node_cfgs, &node_chanmgrs);
+		let description_hash = crate::Sha256(Hash::hash("Testing description_hash".as_bytes()));
+		let invoice = ::utils::create_invoice_from_channelmanager_with_description_hash_and_duration_since_epoch(
+			&nodes[1].node, nodes[1].keys_manager, Currency::BitcoinTestnet, Some(10_000),
+			description_hash, Duration::from_secs(1234567),
+		)
+		.unwrap();
+		assert_eq!(invoice.amount_pico_btc(), Some(100_000));
+		assert_eq!(
+			invoice.min_final_cltv_expiry(),
```

Comment:
No need to break this into multiple lines, the line written out isn't very long.